DeCYFIR- Files Updates (#24136)
demisto/content: Packs/DeCYFIR/ReleaseNotes/1_0_1.md

#### Integrations

> ##### DeCYFIR
> 
> - Updated content in  README.md, decyfir_description.md and Pack-ignore files.

Reviewer: ```suggestion
##### DeCYFIR
- Updated the integration description and README file.
```
